nifi-extension-bundles/nifi-azure-bundle/pom.xml

```diff
@@ -29,7 +29,9 @@
     <properties>
         <!-- when changing the Azure SDK version, also update msal4j to the version that is required by azure-identity -->
         <azure.sdk.bom.version>1.2.25</azure.sdk.bom.version>
-        <msal4j.version>1.15.1</msal4j.version>
+        <!-- msal4j:1.16.x compatible with msal4j:1.14, msal4j:1.15 APIs -->
+        <!-- https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/828 -->
```

Comment:
Although these comments are helpful now, they could quickly become outdated on subsequent updates.

Reply:
Sure, please remove the comment if you think it is better.

Reply:
https://github.com/apache/nifi/pull/9104

This PR covers the same version in 1.x.  Would you mind having a look there as well?

Reply:
... same guidance on the comment.  I think that contributor was following my lead.

Reply:
Thanks @greyp9, I will remove the comments and merge this version update to the main branch. I recommend doing the same on the other PR, feel free to proceed with one if it otherwise looks good to you.